node/node.go

```diff
@@ -488,6 +559,52 @@ func createTransport(
 	)
 }
 
+func createPeerManager(config *cfg.Config, p2pLogger log.Logger, nodeID p2p.NodeID) (*p2p.PeerManager, error) {
+	options := p2p.PeerManagerOptions{
+		MaxConnected:           64,
```

Comment:
will this be configurable in the future?

Reply:
Yes, these will come from configuration :)

Reply:
These are mainly just placeholders I believe to get things running in e2e tests.